docs/manual/mod/mod_dav.xml

```diff
@@ -194,6 +194,36 @@ Alias "/php-source" "/home/gstein/php_files"
 </usage>
 </directivesynopsis>
 
+<directivesynopsis>
+<name>DavBasePath</name>
+<description>Configure repository root path</description>
+<syntax>DavBasePath <var>root-path</var></syntax>
+<default>DavBasePath /repos/</default>
```

Comment:
Maybe

```suggestion
<default>None</default>
```

Reply:
Thanks for catching this, fixed in 59c725a